.circleci/config.yml

```diff
@@ -16,7 +19,7 @@ jobs:
             - ruby-cache-
 
       # Bundle install dependencies
-      - run: gem install bundler --no-document
+      - run: gem install bundler -v 1.17.3 --no-document
```

Comment:
Interesting that this is necessary. Is this a Jenkins limitation?

Reply:
More of a question of alignment between the gemspec and the ci config. I think the gemspec still has ~> 1.x for bundler as a dev dependency. Something that could easily be changed later. Without a version the latest 2.x is installed.